packages/snaps-sdk/src/jsx/components/Value.ts

```diff
@@ -0,0 +1,36 @@
+import { createSnapComponent } from '../component';
+
+/**
+ * The props of the {@link Value} component.
+ *
+ *
+ */
+export type ValueProps = {
+  value: string;
+  extra: string;
```

Comment:
Should this be optional?

Reply:
The underlying component isn't, but it probably could be? https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/confirm/info/row/value-double.tsx#L15-L16

Reply:
I guess there's no use case for it at the moment. We can reconsider when we accept different kinds of props or values.

Reply:
We can always make it optional later yeah